src/sentry/static/sentry/app/components/scoreCard.tsx

```diff
@@ -13,62 +13,51 @@ type Props = {
   score?: React.ReactNode;
   help?: React.ReactNode;
   trend?: React.ReactNode;
-  trendStyle?: 'good' | 'bad';
+  trendStatus?: 'good' | 'bad';
   className?: string;
 };
 
-function ScoreCard({title, score, help, trend, trendStyle, className}: Props) {
+function ScoreCard({title, score, help, trend, trendStatus, className}: Props) {
   return (
     <StyledPanel className={className}>
-      <PanelBody withPadding>
-        <TitleWrapper>
-          <Title>{title}</Title>
-
-          {help && <StyledQuestionTooltip title={help} size="sm" position="top" />}
-        </TitleWrapper>
-
-        <ScoreWrapper>
-          <Score>{score ?? '\u2014'}</Score>
-
-          {defined(trend) && <Trend trendStyle={trendStyle}>{trend}</Trend>}
-        </ScoreWrapper>
-      </PanelBody>
+      <HeaderTitle>
+        <Title>{title}</Title>
+        {help && <QuestionTooltip title={help} size="sm" position="top" />}
+      </HeaderTitle>
+
+      <ScoreWrapper>
+        <Score>{score ?? '\u2014'}</Score>
+        {defined(trend) && <Trend trendStatus={trendStatus}>{trend}</Trend>}
+      </ScoreWrapper>
     </StyledPanel>
   );
 }
 
 function getTrendColor(p: TrendProps & {theme: Theme}) {
-  switch (p.trendStyle) {
+  switch (p.trendStatus) {
     case 'good':
       return p.theme.green300;
     case 'bad':
       return p.theme.red300;
     default:
-      return p.theme.blue300;
+      return p.theme.gray300;
   }
 }
 
 const StyledPanel = styled(Panel)`
-  margin-bottom: 0;
-  min-height: 105px;
+  padding: ${space(2)} ${space(3)};
+  min-height: 100px;
```

Comment:
The `105px` was there to prevent the jumping while loading:

https://user-images.githubusercontent.com/9060071/105954661-50ebc180-6075-11eb-9f7e-43d018747285.mp4

Reply:
Ah, okay - I'll change that back.

Reply:
Wait I don't see that problem @matejminar 
![scorecard](https://user-images.githubusercontent.com/4830259/106033122-353ce700-6086-11eb-992e-ad31e659a648.gif)

Reply:
@doralchan It's only happening in firefox.

Reply:
Ok - I made some minor tweaks! Take a look @matejminar 

Firefox:
![firefox](https://user-images.githubusercontent.com/4830259/106049239-c918ae00-609a-11eb-9f66-38469f357736.gif)

Chrome: 
![chrome](https://user-images.githubusercontent.com/4830259/106049222-c6b65400-609a-11eb-8ae4-54141f8765c1.gif)

Reply:
Sorry for being PITA but this is driving me crazy:

https://user-images.githubusercontent.com/9060071/106057197-e62b9c00-60ef-11eb-8b74-c60df7b327e5.mp4